src/NetworkTest/testQuality/helpers/subscriberMOS.ts

```diff
@@ -18,6 +18,8 @@ const calculateBitRate = (type: AV, current: OT.SubscriberStats, last: OT.Subscr
   return current[type] && current[type].bytesReceived ?
     (8 * (current[type].bytesReceived - last[type].bytesReceived)) / (interval / 1000) : 0;
 };
+const MS_PER_SEC = 1000;
+const DEFAULT_DELAY = 150; // expressed in ms
```

Comment:
Added this default delay based on https://github.com/opentok/gru/blob/develop/src/main/java/com/tokbox/gru/extractors/AudioScoreAlgorithm.java#L7